garage/logger/outputs.py

```diff
@@ -0,0 +1,121 @@
+"""Contains the output classes for the logger.
+
+Each class is sent logger data and handles it itself.
+"""
+import abc
+import csv
+import datetime
+import os
+import sys
+
+import dateutil.tz
+
+from garage.logger.tabular_input import TabularInput
+from garage.misc.console import colorize, mkdir_p
+
+
+class LogOutput(abc.ABC):
+    """Abstract class for Logger Outputs."""
+
+    @property
+    def types_accepted(self):
```

Comment:
Hmm `types_accepted()` or `input_types_accepted()`? @naeioi @krzentner

Reply:
I think it's fine as is.